docs/2.8.0/versions.json

```
@@ -1,5 +1,5 @@
 {
-  "daml": "2.7.0-snapshot.20230719.11983.0.vd999a21a",
+  "daml": "cbcaca974ac24c69246c08f167f20b759ebfade6",
```

```suggestion
  "daml": "2.8.0-snapshot.20230912.12110.0.v788fccd9",
```
These must match folders on Artifactory, so using git commits won't work here.

---

ah, I didn't see this last week and thought I had to wait until a snapshot _release_, now I see that the snapshots on Artifactory are enough. Oh well, lesson learned for the next time :)